api/config/config.go

```diff
@@ -315,15 +301,15 @@ type ModelClientKeepAliveConfig struct {
 type StandardTransformerConfig struct {
 	ImageName             string `validate:"required"`
 	FeastServingURLs      FeastServingURLs
-	FeastCoreURL          string                       `validate:"required"`
-	FeastCoreAuthAudience string                       `validate:"required"`
-	EnableAuth            bool                         `default:"false"`
-	FeastRedisConfig      *FeastRedisConfig            `default:"-"`
-	FeastBigtableConfig   *FeastBigtableConfig         `default:"-"`
-	FeastGPRCConnCount    int                          `validate:"required" default:"10"`
-	FeastServingKeepAlive *FeastServingKeepAliveConfig `default:"-"`
-	ModelClientKeepAlive  *ModelClientKeepAliveConfig  `default:"-"`
```

Comment:
@krithika369 I've removed the tags from these two configs is never expected to be `nil`. This also allows their nested `default` tags to work and set the correct default values on them.